### PR TITLE
Code Owners update respecting each team

### PR DESCRIPTION
**Pull request by NickyMateev**
# Description

This PR sets specific restrictions on ownership of directories with respect to our teams - dragon, falcon, raptor.

### CODE REVIEW

File: chart/compass/templates/filters/OWNERS [hunk 1]
```diff
@@ -0,0 +1,6 @@
+reviewers:
```
Comment (desislavaa) [inline chart/compass/templates/filters/OWNERS:1]: raptor? 🙃

**Review by DimitarPetrov — APPROVED** [review]

**Pull request merged**